src/main/java/me/ryanhamshire/GriefPrevention/GriefPrevention.java

```diff
@@ -582,7 +585,10 @@ else if(world.getEnvironment() == Environment.NORMAL)
         String accessTrustSlashCommands = config.getString("GriefPrevention.Claims.CommandsRequiringAccessTrust", "/sethome");
         this.config_claims_supplyPlayerManual = config.getBoolean("GriefPrevention.Claims.DeliverManuals", true);
         this.config_claims_manualDeliveryDelaySeconds = config.getInt("GriefPrevention.Claims.ManualDeliveryDelaySeconds", 30);
-        
+
+		this.config_claims_firespreads = config.getBoolean("GriefPrevention.Claims.FireSpreadsInClaims", false);
+		this.config_claims_firedamages = config.getBoolean("GriefPrevention.Claims.FireDamagesInClaims", false);
```

Comment:
Looks like tabs or something happened here.

(I know, the project is a mess of tabs and spaces, but we want to at least attempt to keep some consistency instead of making it worse.)

Reply:
oh shoot how did i not see that!

Reply:
okay fixed